api/database/migrations/20191111203354-add-promo-codes.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface
+        .addColumn("HackerProfiles", "promoCode", {
+          type: Sequelize.STRING(100)
```

Comment:
we want promoCode to be unique right? prob want to add the unique attribute to this field https://stackoverflow.com/questions/28116187/unique-constraint-on-sequelize-column

Reply:
true, will fail in the off chance that someone generates a duplicate but should be fine...